examples/graphql-react-apollo/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "graphql-react-apollo",
   "version": "0.1.0",
-  "private": true,
```

Comment:
Removing the `private` key from workspaces for them to be listed when run `yarn workspaces info`.
If a workspace contains `private: true`, it will get ignored when prompted for the list of all workspaces.